lib/index.js

```diff
@@ -183,9 +185,9 @@ Builder.prototype.build = function(options) {
 			// Keep filename for later usage (see ImportCollectorPlugin) as less modifies the parserOptions.filename
 			var filename = parserOptions.filename;
 
-			// Only use custom file handler when "rootPaths" are defined
+			// Only use custom file handler when "rootPaths" or custom "fs" are used
 			var fnFileHandler;
-			if (options.rootPaths && options.rootPaths.length > 0) {
+			if (options.rootPaths && options.rootPaths.length > 0 || that.customFs) {
```

Comment:
Maybe add brackets to make the condition easier to read

Reply:
Done 👍